test/TestUtilities/ApproxArray.hpp

```diff
@@ -0,0 +1,72 @@
+#ifndef SFML_TESTUTILITIES_APPROXARRAY_HPP
+#define SFML_TESTUTILITIES_APPROXARRAY_HPP
+
+#include <doctest.h>
+#include <array>
+
+namespace std
+{
+    template <typename T, std::size_t size>
+    std::ostream& operator<<(std::ostream& out, const std::array<T, size>& array)
+    {
+        if (size == 0)
+            return out << "{}";
+
+        out << "{ ";
+        for (size_t i = 0; i + 1 < array.size(); ++i)
+            out << array[i] << ", ";
```

Comment:
There's a tiny inconsistency here with the use of `size_t` instead of `std::size_t`.